ui-admin/src/forms/designer/utils/formDraftUtils.tsx

```diff
@@ -0,0 +1,42 @@
+import { VersionedForm } from '@juniper/ui-core'
+
+export type FormDraft = {
+  content: string
+  date: number
+}
+
+/** returns a form draft key for local storage */
+export function getFormDraftKey({ form }: { form: VersionedForm }) {
+  return `surveyDraft_${form.id}_${form.version}`
```

Comment:
note to self: minor detail, but rename this and a few other places to `formDraft_...` for consistency and correctness

Reply:
I would also construct the key as form.stableId + form.version so that it's more human-readable, as opposed to using the id